### ISSUE TITLE
Send Valentine's Day Card To Your Someone Special

### ISSUE THREAD

**Issue by cosmeagardens** [cosmeagardens/CosmeaGardens#1]
Since tradition, sending Valentine cards was meant to express love on this special day from your dear ones. Most were hand-crafted and sent along with [love flowers](https://www.cosmeagardens.com/) such as roses and daisies. As of today, we will find varieties of Valentine's cards, from cheeky, humorous, and romantic types. One can make it more personal by DIY. It gives you a chance to use your creativity as you think about that someone special.
![image](https://user-images.githubusercontent.com/59050914/74004659-6ebfa280-499c-11ea-8836-029b784936de.png)

Customize the Card
When you love someone, you want to show it by giving a special card with a love message that expresses how much you love them. One way to achieve this is by buying a valentine's card from the gift card shop. Unfortunately, the cards sold might not convey the appropriate message that you want. Customization becomes the next best option.

Through online, you can create Valentine's card for your sweetheart, using a variety of styles available. Upload your most memorable romantic photo on the card you are creating. Write a message that will speak to your love and then pay a small amount to have it printed professionally. It will be a more significant card, unlike the one randomly bought from the shop.

Teach the Children Too
Although some might not agree on this, they may dispute that Valentine is ostensibly for lovers. It is good to teach the children the traditions of Valentine's cards from an early stage. It becomes part of the history lesson and will also enhance the creativity of crafting cards and showing each other mutual love. Which also proves the right way of introducing gift exchange.

Although some might not agree on this, they may dispute that Valentine is ostensibly for lovers. It is good to teach the children the traditions of Valentine's cards from an early stage. It becomes part of the history lesson and will also enhance the creativity of crafting cards and showing each other mutual love. Which also proves the right way of introducing gift exchange.

Send the Flowers and Pin a Card
Ever felt like sending love flowers might not be enough to hammer your message? A tip for you here; add a card that you have personalized. It articulates your message to that special someone and eliminates the confusion you might have over a person that you have just met, and you happen to like.

Do It Yourself (DIY)

For those who are not crazy about digital can hand-craft a card for their love. There are so many DIYs of achieving this. You can make a cute and flirty design card that you know will warm your loved one. Get cocky with decoration. Add lovely photos that will have your partner drooling over them.

In conclusion, whether you are buying your loved one a card from the stores, online, or hand-crafting it, make it sentimental. Send love flowers to go along with the card. Don't feel confused as we have sampled ways to make it easier for you in this article. Have fun and send that valentine's card to brighten that special someone's day.
